koptional/src/main/kotlin/com/gojuno/koptional/Optional.kt

```diff
@@ -6,6 +6,8 @@ sealed class Optional<out T : Any> {
         is Some -> value
         is None -> null
     }
+
+    open operator fun component1() = toNullable()
```

Comment:
Let's also specify return type explicitly here, that makes it easier for users when they look at sources